.changeset/eight-webs-buy.md

```diff
@@ -0,0 +1,18 @@
+---
+'@tanstack/query-broadcast-client-experimental': patch
+'@tanstack/query-async-storage-persister': patch
+'@tanstack/angular-query-persist-client': patch
+'@tanstack/query-sync-storage-persister': patch
+'@tanstack/svelte-query-persist-client': patch
+'@tanstack/angular-query-experimental': patch
+'@tanstack/react-query-persist-client': patch
+'@tanstack/solid-query-persist-client': patch
+'@tanstack/query-persist-client-core': patch
+'@tanstack/svelte-query': patch
+'@tanstack/react-query': patch
+'@tanstack/solid-query': patch
+'@tanstack/query-core': patch
+'@tanstack/vue-query': patch
```

Comment:
I think it’s enough to put `query-core` here, as it will release the others automatically as they are a dependency.